src/components/OutdatedMicrobitWarning.svelte

```diff
@@ -5,24 +5,23 @@
  -->
```

Comment:
I think we should delete this file for now.

Reply:
This depends on if we care about the text/translations. Otherwise we could gut the logic and keep the html?

Reply:
We've deleted it but kept the translations for now (exemption in test)